chaoslib/pumba/network_chaos/network_chaos.yml

```diff
@@ -86,13 +78,19 @@
           engine_ns: "{{ c_ns }}"
           message: "Injecting {{ c_experiment }} chaos on {{ app_pod }} pod"
         when: "c_engine is defined and c_engine != ''"
-  
-      - name: Setup pumba chaos infrastructure
-        shell: >
-          kubectl create -f /chaoslib/pumba/network_chaos/pumba_kube.yml -n {{ a_ns }}
-        args: 
-          executable: /bin/bash
-        register: pumba_deploy_result
+
+       # Pumba pods creation is attempted for a total of 3 times, if it is not immediately schedulable due to transient node conditions
+       # If the pumba pod is not schedulable across these 3 tries, the experiment is failed with message indicating improper cluster state.
+      - include_tasks: /utils/common/create_chaos_pod.yml
```

Comment:
where is this file?

Reply:
This file is in #1378 PR, That's why added don't merge..

Reply:
ok